wallet/storage/src/internal/store_tx.rs

```diff
@@ -52,13 +55,53 @@ mod well_known {
     }
 
     declare_entry!(StoreVersion: u32);
+    declare_entry!(EncryptionKeyKdfChallenge: KdfChallenge);
+}
+
+#[derive(PartialEq, Clone)]
+pub enum EncryptionState {
+    // The DB is encrypted and we don't have the key to decrypt it
```

Comment:
I wouldn't say "The DB"... I would say "The secret parts of the DB, such as private keys". The distinction is vital, because if the whole DB is encrypted, then nothing is accessible, which isn't true based on our design.

Reply:
fixed